src/components/Search/SearchInput.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+
+import Grid from '@mui/material/Grid';
+import TextField from '@mui/material/TextField';
+import { connect } from 'react-redux';
+
+import store from '../../common/store';
+import { fetchRecentRevisions } from '../../thunks/searchThunk';
+import { Revision, State } from '../../types/state';
+import { handleChangeSearch } from '../../utils/searchViewHelper';
+import SearchResultsList from './SearchResultsList';
+
+class SearchInput extends React.Component<SearchInputProps> {
+  componentDidMount() {
+    const repo = store.getState().search.repository;
+    store.dispatch(fetchRecentRevisions(repo));
+  }
+
+  render() {
+    const {
+      focused,
+      handleFocus,
+      handleChildClick,
```

Comment:
`focused` `handleFocus` and `handleChildClick` are for detecting if results should be hidden using Julien's suggestion from #89